src/components/common/NameRequestInfo.vue

```diff
@@ -16,14 +16,14 @@
           </li>
```

Comment:
Cleanup, especially because I deleted parseNameRequest() from the mixin.

I think this is purer/cleaner/simpler.